test/Microsoft.IdentityModel.Tests/Default.cs

```diff
@@ -191,6 +196,21 @@ public static string OriginalIssuer
             get { return "http://Default.OriginalIssuer.com"; }
         }
 
+        public static string Role
+        {
+            get { return "role"; }
+        }
+
+        public static string NameSpace
```

Comment:
spelling: NameSpace -> Namespace (it is a single word).

Is this specific to SamlAttribute? If so, use ClaimNamespace of SamlAttributeClaimNamespace